db-commons/src/main/scala/org/bitcoins/db/AppConfigFactory.scala

```diff
@@ -1,13 +1,18 @@
 package org.bitcoins.db
 
-import java.nio.file.Path
+import java.nio.file.{Path, Paths}
 
 import com.typesafe.config.Config
 
 import scala.concurrent.ExecutionContext
 
 trait AppConfigFactory[C <: AppConfig] {
 
+  def fromConfig(config: Config)(implicit ec: ExecutionContext): C = {
```

Comment:
We should add 

```scala

def fromClassPathConfig()(impliicit ec: ExecutionContext): C = { 
  fromConfig(ConfigFactory.load())
}
```

^ That needs to be unit tested, it can probably be unit tested in the same way we test things here: https://github.com/bitcoin-s/bitcoin-s/blob/475ed19df6634f584ce828ac597ca54704963e8b/db-commons-test/src/test/scala/org/bitcoins/db/DBConfigTest.scala#L24 

However I think the path will need to be adjusted to be inside of 

`src/test/resources/application.conf`

You should assert that the datadir (and maybe a few other random settings in different projects) are set correctly.

Reply:
More typesafe documentation: https://github.com/lightbend/config#standard-behavior